src/gridstack-poly.js

```diff
@@ -202,4 +219,46 @@ if (!Array.prototype['forEach']) {
     }
     // 8. return undefined
   };
-}
\ No newline at end of file
+}
+
+// https://developer.mozilla.org/zh-CN/docs/Web/API/CustomEvent/CustomEvent

```

Comment:
small request, can you use en-US in urls instead ? more people read english doc. thank you.